dracut/modules.d/99kiwi-dump-reboot/kiwi-dump-reboot-system.sh

```diff
@@ -33,17 +33,36 @@ function boot_installed_system {
     fi
 }
 
+function mount_ramdisk_sysroot {
+    local image_target
+    local root_uuid
+
+
+    [ -e /config.bootoptions ] || die "Missing config.bootoptions file"
+
+    root_uuid=$(
+        while read -r -d ' ' opt; do echo "${opt}";done < /config.bootoptions |\
+        grep root= | cut -f2- -d=
+    )
+
+    [ -z "${root_uuid}" ] && \
+        die "'root=' argument not found in config.bootoptions"
+
+
+    image_target=$(get_selected_disk)
+    kpartx -s -a "${image_target}"
```

Comment:
One thing I'd like to mention is that the method could reduce duplication a bit and avoid double empty lines, e.g

```bash
function mount_ramdisk_sysroot {
￼    local image_target
￼    local root_uuid
    local boot_options=/config.bootoptions
￼
￼    if [ ! -e "${boot_options}" ]; then
        die "Missing ${boot_options} config file"
    fi
￼    root_uuid=$(
￼        while read -r -d ' ' opt; do echo "${opt}";done < "${boot_options}" |\
￼        grep root= | cut -f2- -d=
￼    )
￼    if [ -z "${root_uuid}" ];then
￼        die "'root=' argument not found in ${boot_options}"
    fi
￼    image_target=$(get_selected_disk)
￼    kpartx -s -a "${image_target}"
```

I'm suggesting this because in case of an error the debug information is a nicer read. Thoughts ?